packages/block-library/src/file/edit.js

```diff
@@ -182,7 +183,7 @@ class FileEdit extends Component {
 				</BlockControls>
 				<div className={ classes }>
 					<div className={ `${ className }__content-wrapper` }>
-						<RichText
+						<PlainText
 							wrapperClassName={ `${ className }__textlink` }
```

Comment:
Not all of these props are supported by PlainText. They should be updated or removed accordingly.

https://github.com/WordPress/gutenberg/tree/master/packages/block-editor/src/components/plain-text